src/routes/admin/_admin.transactions.tsx

```diff
@@ -62,6 +62,8 @@ export function Transactions() {
   const [debouncedBlockHeight] = useDebounce(blockHeight, 200);
   const { order_by_field, sort_direction } = useSearch({ from: '/admin/_admin/transactions' });
 
+  const [showRecordTransaction] = useState(false);
```

Comment:
You don't need state here.

Reply:
Sure. I thought that maybe we will want to `setShowRecordTransaction` in the future. But agree. I will leave just a simple boolean flag.